### PR TITLE
Replace stdout / stderr with commons-logging

### PR DESCRIPTION
**Pull request by svogt**
Use logging instead of stdout / stderr

### CODE REVIEW

File: vfs2provider-jdbctable/src/main/java/com/seeburger/vfs2/provider/jdbctable/JdbcTableRowFile.java [hunk 1]
```diff
@@ -1008,7 +1011,7 @@ private void rollbackConnection(Blob blob, ResultSet rs, PreparedStatement ps, C
             }
             catch (Exception e)
             {
-                e.printStackTrace(System.err);
+                LOG.warn(e);
```
Comment (ecki) [inline vfs2provider-jdbctable/src/main/java/com/seeburger/vfs2/provider/jdbctable/JdbcTableRowFile.java]: I would use („Ignoring exception while cleaning up connection“, e)“ or a similar message.

File: vfs2provider-jdbctable/src/main/java/com/seeburger/vfs2/provider/jdbctable/JdbcDialectBase.java [hunk 2]
```diff
@@ -22,6 +22,9 @@
 
 import javax.sql.DataSource;
 
+import org.apache.commons.logging.Log;
```
Comment (ecki) [inline vfs2provider-jdbctable/src/main/java/com/seeburger/vfs2/provider/jdbctable/JdbcDialectBase.java:25]: Yes, seems ok, commons logging is used in VFS as well.

**Review by ecki — CHANGES_REQUESTED** [review]
Tanks, looks good. I would just add a log message to the ignored exception while we are at it.

File: vfs2provider-jdbctable/src/main/java/com/seeburger/vfs2/provider/jdbctable/JdbcTableRowFile.java (hunk 1 above)
Comment (svogt) [reply vfs2provider-jdbctable/src/main/java/com/seeburger/vfs2/provider/jdbctable/JdbcTableRowFile.java]: done

**Review by ecki — APPROVED** [review]
Danke, sieht gut aus.

**Pull request merged**